tests/methodologies/johnny_decimal/test_categories.py

```
@@ -508,3 +510,59 @@ def test_default_areas_defined(self):
+        with pytest.raises(ValueError, match="Category must be 0-99"):
+            CategoryDefinition(area=10, category=100, name="Bad", description="")
+
+    # Line 234-235: CategoryDefinition.__post_init__ — empty name
+    def test_category_def_empty_name(self) -> None:
+        with pytest.raises(ValueError, match="name cannot be empty"):
+            CategoryDefinition(area=10, category=1, name="", description="")
+
+    # Line 282: NumberingResult.__post_init__ — file_path as string
+    def test_numbering_result_string_path(self) -> None:
+        result = NumberingResult(
+            file_path="/tmp/test.txt",  # type: ignore[arg-type]
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.txt"

(S108)